pilot/pkg/networking/plugin/registry/registry.go

```diff
@@ -20,13 +20,14 @@ package registry
 import (
 	"istio.io/istio/pilot/pkg/networking/plugin"
 	"istio.io/istio/pilot/pkg/networking/plugin/authn"
+	"istio.io/istio/pilot/pkg/networking/plugin/mixer"
 )
 
-// NewPlugins returns a list of plugin instance handles. Each plugin implements the plugin.Callbacks interfaces
-func NewPlugins() []plugin.Plugin {
-	plugins := make([]plugin.Plugin, 0)
+// NewDefaultPlugins returns a slice of default Plugins.
+func NewDefaultPlugins() []plugin.Plugin {
+	var plugins []plugin.Plugin
```

Comment:
name change and return literal: done (there was previously a NewPlugins that did something else).
regarding NewPlugins->Plugins, it's more conventional to name constructors NewType (i've unsuccessfully argued in that past that MakeType is often more appropriate). it seems to be the de facto naming choice for better or worse.

Reply:
That's fine; actually I don't like `New*`, it's a java-ism that was imported into our codebase by people new to go. But, since that is the defacto standard in the codebase, I think consistency is more important than "correctness" (for whatever correctness is in this fairly subjective case).